### PR TITLE
13 issue

### PR DESCRIPTION
**Pull request by rahulcs754**
- reset my position 
- created class icon-l (icon large)  , icon-m(icon medium) , icon-s(icon small) 
image

Please check and provide your feedback

### CODE REVIEW

File: css/components/badge.css [hunk 1]
```diff
@@ -53,4 +53,17 @@
...
+    font-size: 3rem;
+}
+
+.icon-m{
+    font-size: 2rem;
+}
+
+.icon-s{
+    font-size: 1rem;
+}
```
Comment (shubhamkumar648) [inline css/components/badge.css:69]: all variable looks good to me!

**Review by shubhamkumar648 — APPROVED** [review]
you can merge

File: css/components/badge.css (hunk 1 above)
Comment (vaibhav-mahalle) [reply css/components/badge.css:69]: you can use sm instead of s for better readability.

**Review by vaibhav-mahalle — COMMENTED** [review]
suggested some changes. please do the changes.

File: css/components/badge.css (hunk 1 above)
Comment (rahulcs754) [reply css/components/badge.css:69]: made the required changes please look into it and approve the changes.

**Review by vaibhav-mahalle — APPROVED** [review]
try to use meaningful class names. also try to reuse the code wherever possible.

**Pull request merged**